pkgs/by-name/re/recaf/package.nix

```diff
@@ -0,0 +1,107 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  git,
+  jdk23,
+  gradle,
+  makeWrapper,
+  makeDesktopItem,
+  copyDesktopItems,
+}:
+let
+  pname = "recaf";
+  version = "4.0.0-SNAPSHOT";
```

Comment:
What does this version mean? Couldn't find the tag

Reply:
There is currently no stable release. The project has a snapshot (unstable) version https://github.com/Col-E/Recaf/blob/40ecd657bc52670789f59d7c5fe266fa5940c6ef/build.gradle#L9

Reply:
What about these tags? https://github.com/Col-E/Recaf/tags Do they work properly?

Reply:
These tags are for an older version that is stable but no longer supported. There is a PR for Recaf 2.x that was created later - https://github.com/NixOS/nixpkgs/pull/354184. The current master branch is 4.0.0

How should we figure it out? Should I rename the package to recaf4 or recaf_4 so that the stable package (version 2) will be called recaf or leave it as it is